data_store/table_extraction/process.py

```diff
@@ -75,38 +76,45 @@ def process(self, table: Table) -> None:
         :param table: The table to process.
         :return: None
         """
+
         self._strip_whitespace(table)
         self._lift_header(table)
         self._remove_merged_headers(table)
         self._drop_cols_by_name(table)
         self._remove_ignored_non_header_rows(table)
         self._replace_dropdown_placeholder(table)
         self._drop_bespoke_rows(table)
+
         if self.drop_empty_rows:
             self._drop_empty_rows(table)
+
         if self.drop_empty_tables and table.df.empty:
-            table.df = None
+            table.df = pd.DataFrame()
 
     def _lift_header(self, table: Table) -> None:
-        header = table.df.iloc[: self.num_header_rows]
-        header = self._concatenate_headers(header, headers_to_ffill=self.merged_header_rows)
-        table.df.columns = header
+        header: DataFrame = table.df.iloc[: self.num_header_rows]
+        table.df.columns = Index(
```

Comment:
table.df.columns accepts only an Index type. Easily fixed by passing the list to the `Index`.